configuration/bin/startup.sh

```diff
@@ -44,6 +45,11 @@ while [[ $# -gt 0 ]]; do
       shift
       shift
       ;;
+    -fn)
+      config_filename="$2"
+      shift
+      shift
+      ;;
     -maxheapsize)
       MAX_HEAP_SIZE="$2"
```

Comment:
the params after config_filename need to update ${number}

Reply:
eg. MAX_HEAP_SIZE="$3"

Reply:
same